index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Maher</title>
+    <link rel="stylesheet" href="css/style.css">
+    <!-- link swiper css -->
+    <link rel="stylesheet" href="CSS/swiper-bundle.min.css">
+    <!-- BOX ICONS -->
+</head>
+<body>
+    <!-- Header -->
+        <!-- nav -->
+        <div class="nav container">
+            <!-- Logo -->
+            <div class="navbar">
+            <p  href="#">Maher</p>
+          </div>
+          <div class="end">
+            <img class="endnavbar" src="./img/Union.png" alt="Menu">
+          </div>
+          </div>
+     <!-- Home -->
+        <div class="home-content">
+          <h1>Maher Ramadan</h1>
+        </div>
+            <div class="home-text">
+              <h2>Software Developer</h2>
+            </div>  
+            <div class="paragraph">
+              <p>I can help you build a product,
+                feature or website Look through 
+                some of my work and experience! If  you like what you see 
+                and have a project that  needs coding,  don’t
+                 hesitate to  contact me. </p>
+            </div>
+            <div class="hr" style="padding:20px; margin: 20px; color: dark">
+              <hr>
+            </div>
+    <!-- line1 -->
+<div class="third">
+    <div class="languages">
+      <h4>Languages</h4>
+      <div class="menu-language">
+        <span>JavaScript</span>
+        <span>Ruby</span>
+        <span>Html</span>
+        <span>CSS</span>
+      </div>
+    </div>
+
```

Comment:
- [ ] Kindly use `<ul>` and `<li>` to group and list your technologies. This is also important for semantic and accessibility reasons.